test/Microsoft.ML.StaticPipelineTesting/Microsoft.ML.StaticPipelineTesting.csproj

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.HalLearners\Microsoft.ML.HalLearners.csproj" />
```

Comment:
Yes I will move it at the end of the review so that you can see the old and new file side by side in the meantime.

Reply:
Yes I will move it at the end of the review so that you can see the old and new file side by side in the meantime.

---
In reply to: [229521868](https://github.com/dotnet/machinelearning/pull/1452#discussion_r229521868) [](ancestors = 229521868)